src/vcf/validate_optional_policy.cpp

```diff
@@ -146,6 +152,36 @@ namespace ebi
         }
     }
 
+    void ValidateOptionalPolicy::check_body_entry_alt_gvcf_end(ParsingState & state, Record const & record) const
+    {
+        if (std::find(record.alternate_alleles.begin(), record.alternate_alleles.end(), GVCF) != record.alternate_alleles.end()
+            && record.info.find(END) == record.info.end()) {
+            throw new AlternateAllelesBodyError{state.n_lines,
```

Comment:
I think this would fit better as INFO error, instead of ALT. The function name should be changed as well probably